src/transaction.rs

```diff
@@ -120,7 +120,7 @@ impl Transaction<'_> {
             TransactionBehavior::Immediate => "BEGIN IMMEDIATE",
             TransactionBehavior::Exclusive => "BEGIN EXCLUSIVE",
         };
-        conn.execute(query, NO_PARAMS).map(move |_| Transaction {
+        conn.execute_batch(query).map(move |_| Transaction {
```

Comment:
It's probably better to keep these as `execute` no? Given that it's slightly more efficient?

Reply:
> It's probably better to keep these next few calls as `execute` no? Given that it's slightly more efficient?

My wild guess is: `execute` < `execute_batch` < `sqlite3_exec` for a single statement.

Reply:
Yeah, it's hard for me to imagine it would be anything other than that, so maybe we should keep these parts as using execute()?

Reply:
> Yeah, it's hard for me to imagine it would be anything other than that, so maybe we should keep these parts as using execute()?

I mean `execute_batch` is faster than `execute` but slower than `sqlite3_exec`.
```
test bench_execute       ... bench:       2,923 ns/iter (+/- 291)
test bench_execute_batch ... bench:       2,847 ns/iter (+/- 374)
```

Reply:
Huh, very strange, fair enough though, nevermind!

(I'm probably going to dig into this later since it means `execute` with `NO_PARAMS` is doing work it doesn't have to, but that's for another time)

Reply:
> Huh, very strange, fair enough though, nevermind!
> 
> (I'm probably going to dig into this later since it means `execute` with `NO_PARAMS` is doing work it doesn't have to, but that's for another time)

* check_no_tail
* bind_parameter_count
* check_update
* reset